src/OhmmsApp/RandomNumberControl.cpp

```diff
@@ -258,8 +259,8 @@ void RandomNumberControl::read_old(const std::string& fname, Communicate* comm)
     hout.push("random");
     std::string engname;
     hout.read(slab, Random.EngineName);
-    shape[0] = static_cast<int>(slab.size(0));
-    shape[1] = static_cast<int>(slab.size(1));
+    shape[0]           = static_cast<int>(slab.size(0));
```

Comment:
Looks like a bug in clang-format. No worth fixing the formatting on our side but just wait.

Reply:
@ye-luo Yes, thanks for pointing it out. Let's discuss on slack as we can add a static check for clang-format and settle on a specific version (I'm using 10, but whichever makes more sense for everybody). My IDE has automatic formatting hence there is a few of these. In the meantime, let me undo these clang-format related changes.

Reply:
I approved the PR, namely I would like to keep these formatting. I only encourage people not formatting the source code when a PR touch a lot of files and complicates the review.

Reply:
OK, sounds good!